ecosystem/platform/server/app/javascript/controllers/application.ts

```diff
@@ -34,4 +36,10 @@ const application = Application.start();
 application.debug = false;
 window.Stimulus = application;
 
+Sentry.init({
+  dsn: process.env.SENTRY_DSN,
```

Comment:
We might want to use a separate DSN for the frontend.

Reply:
We do already have the -frontend project set up in Sentry, so I'm updating it to use that.